scripts/start_network.sh

```diff
@@ -26,7 +26,7 @@ sh ${SCRIPT_DIR}/init_relayers.sh STRIDE ${HOST_CHAINS[@]}
 sh ${SCRIPT_DIR}/create_channels.sh ${HOST_CHAINS[@]}
 
 echo "Starting relayers"
-docker-compose up -d hermes icq
+docker-compose up -d hermes # icq
```

Comment:
nit should we just remove icq altogether? or worth keeping as we might need to bring it back if ibc-go breaks?

Reply:
I think this file might be out of date - its removed in the latest commit